gateway-ha/src/main/java/io/trino/gateway/ha/router/TrinoQueryProperties.java

```diff
@@ -142,30 +146,29 @@ public TrinoQueryProperties(
         maxBodySize = -1;
     }
 
-    public TrinoQueryProperties(HttpServletRequest request, boolean isClientsUseV2Format, int maxBodySize)
+    public TrinoQueryProperties(ContainerRequestContext reqContext, boolean isClientsUseV2Format, int maxBodySize)
```

Comment:
I think it is allright to simply use `requestContext` for more descriptive purpose.

Reply:
👍